### PR TITLE
Add a perl section with intro

### PR DESCRIPTION
**Pull request by squain**
**Description**
Add perl section with some introductory lines and further reading

### CODE REVIEW

File: src/pages/perl/index.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+## What is Perl?
+
+Perl is a general-purpose programming language created by Larry Wall in 1987.  It was originally designed for manipulating text.
+
+## What is Perl used for?
+
+It has grown from its original purpose to web development, network programming and more. There are thousands of modules available on the <a href='https://metacpan.org/' target='_blank' rel='nofollow'>CPAN</a> or Comprehensive Perl Archive Network.
+
+Some of the best modules are listed in the <a href='https://metacpan.org/pod/Task::Kensho' target='_blank' rel='nofollow'>Task::Kensho module</a>.
```
Comment (jonathan-grah) [inline src/pages/perl/index.md]: Please use [markdown style links](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md#links).

File: src/pages/perl/index.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+title: Perl
+---
+
+## What is Perl?
+
+Perl is a general-purpose programming language created by Larry Wall in 1987.  It was originally designed for manipulating text.
+
+## What is Perl used for?
+
+It has grown from its original purpose to web development, network programming and more. There are thousands of modules available on the <a href='https://metacpan.org/' target='_blank' rel='nofollow'>CPAN</a> or Comprehensive Perl Archive Network.
```
Comment (jonathan-grah) [inline src/pages/perl/index.md]: Please use [markdown style links](https://github.com/freeCodeCamp/guides/blob/master/CONTRIBUTING.md#links).

File: src/pages/perl/index.md [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+
+Some of the best modules are listed in the <a href='https://metacpan.org/pod/Task::Kensho' target='_blank' rel='nofollow'>Task::Kensho module</a>.
+
+## Further Reading
```
Comment (jonathan-grah) [inline src/pages/perl/index.md]: Please rename to "Resources" and use heading 3 (`###`).

File: src/pages/perl/index.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
+---
+title: Perl
+---
+
+## What is Perl?
```
Comment (jonathan-grah) [inline src/pages/perl/index.md]: Please use heading 3 (`###`).

File: src/pages/perl/index.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
...
+
+Perl is a general-purpose programming language created by Larry Wall in 1987.  It was originally designed for manipulating text.
+
+## What is Perl used for?
```
Comment (jonathan-grah) [inline src/pages/perl/index.md]: Please use heading 3 (`###`).

**Comment by domwakeling** [comment]
Marked as stale and closed due to inactivity. :disappointed:

Please make appropriate changes if you have the time

**Pull request closed without merging**